docs/operators/http_input.md

```diff
@@ -14,7 +14,7 @@ The `http_input` operator starts an HTTP server for receiving log messages.
 | `max_header_size`  | 1mb                   |                                        |
 | `max_body_size`    | 10mb                  |                                        |
 | `auth`             |                       | An optional `Auth` configuration (see the Auth configuration section)               |
-| `tls`              |                       | An optional `TLS` configuration (see the TLS configuration section)               |
+| `tls`              | nil                   | An optional `TLS` configuration (see the TLS configuration section). |
```

Comment:
Nit: This isn't really a user facing concern. I think omitting it communicates the meaning that the user is concerned with.
```suggestion
| `tls`              |                   | An optional `TLS` configuration (see the TLS configuration section). |
```